bitnami/keycloak/templates/secrets.yaml

```
@@ -11,8 +11,8 @@ metadata:
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/component: keycloak
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- if or .Values.commonAnnotations .Values.auth.annotations }}
+  annotations: {{- include "common.tplvalues.merge" ( dict "values" (list .Values.commonAnnotations .Values.auth.annotations) "context" $ ) | nindent 4 }}
```

Same suggestion than above

```suggestion
  annotations: {{- include "common.tplvalues.merge" ( dict "values" (list .Values.auth.annotations .Values.commonAnnotations) "context" $ ) | nindent 4 }}
```

---

Fixed :+1: